centos/6/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Copyright 2015-2016 jitakirin
+#
+# This file is part of docker-rpmbuild.
+#
+# docker-rpmbuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# docker-rpmbuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with docker-rpmbuild.  If not, see <http://www.gnu.org/licenses/>.
+
+# Based on: http://fedoraproject.org/wiki/How_to_create_an_RPM_package
+# And also:
+# - https://registry.hub.docker.com/u/nishigori/rpmbuild
+# - https://registry.hub.docker.com/u/sydneyuni/rpm-build-env/
+#
+# I have forked https://github.com/jitakirin/docker-rpmbuild and modified
+# for my own purposes.
+
+FROM centos:6
```

Comment:
I do wonder if it's better to deal with the code duplication and a one character change, or template mechanism.